sdk/core/test/unit/services/AffinidiVaultStorageService.test.ts

```diff
@@ -0,0 +1,237 @@
+'use strict'
+
+import nock from 'nock'
+import sinon from 'sinon'
+
+import { DidAuthService } from '@affinidi/affinidi-did-auth-lib'
+import { KeysService } from '@affinidi/common'
+
+import AffinidiVaultStorageService from '../../../src/services/AffinidiVaultStorageService'
+import { generateTestDIDs } from '../../factory/didFactory'
+import { testPlatformTools } from '../../helpers/testPlatformTools'
+import { STAGING_AFFINIDI_VAULT_URL, STAGING_REGISTRY_URL } from '../../../src/_defaultConfig'
+import credential from '../../factory/signedCredential'
+import { VaultCredential } from '../../../src/dto/vault.dto'
+import { expect } from 'chai'
+
+let encryptionKey: string
+let encryptedSeed: string
+let audienceDid: string
+const region = 'eu-west-2'
+
+const createAffinidiStorageService = () => {
+  const keysService = new KeysService(encryptedSeed, encryptionKey)
+  const didAuthService = new DidAuthService({ encryptedSeed, encryptionKey })
+  return new AffinidiVaultStorageService(didAuthService, keysService, testPlatformTools, {
+    audienceDid: audienceDid,
+    vaultUrl: STAGING_AFFINIDI_VAULT_URL,
+    accessApiKey: undefined,
+  })
+}
+
+const mockDidAuth = () => {
+  nock(STAGING_AFFINIDI_VAULT_URL)
+    .post('/api/v1/did-auth/create-did-auth-request')
+    .reply(
+      200,
+      '"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyLCJpc3MiOiJkaWQ6ZWxlbTpFaUNILXh4Y25rZ1p2NlF2anZvX1VYbi04RFVkVU4zRXRCSnhvbEFRYlFyQ2NBIyJ9.PC4hlTYT5oc1rtcE3Ngq1LN35vAQXI0QgC2nzzQ9RKw"',
+    )
+
+  nock(STAGING_REGISTRY_URL).post('/api/v1/did-auth/create-did-auth-response').reply(200, {})
```

Comment:
what endpoint is this? the did auth response was generated on the client side, without any request to the server. I remeber we should only make request to server for getting the did auth request token.

Reply:
hm, that what I was found in integration test for revocationList `#buildRevocationListStatus, #revokeCredential`
Anyway without it doesn't working. Basically is it using `DidAuthService` from `@affinidi/affinidi-did-auth-lib`, which actually making call to registry.
Do you have some examples how do that without this call?

Reply:
My bad. It should stay like this. I think in the background this service is called for the flow.